mdstat.go

```diff
@@ -0,0 +1,157 @@
+package procfs
+
+import (
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strconv"
+	"strings"
+	"path"
+
+)
+
+var (
+	statuslineRE = regexp.MustCompile(`(\d+) blocks .*\[(\d+)/(\d+)\] \[[U_]+\]`)
+	buildlineRE  = regexp.MustCompile(`\((\d+)/\d+\)`)
+)
+
+type mdStatus struct {
+	mdName       string
```

Comment:
At least the fields, but I'd export both. Also please add comments to each line (as `golint` would tell you anyways), so that users can understand each field by just looking at godoc.